ios/engine/KMEI/KeymanEngine/Classes/Keyboard/KeymanWebViewController.swift

```diff
@@ -130,6 +130,12 @@ class KeymanWebViewController: UIViewController {
     webView!.backgroundColor = UIColor.clear
     webView!.navigationDelegate = self
     webView!.scrollView.isScrollEnabled = false
+    
+    if #available(iOSApplicationExtension 16.4, *) {
+      if(Version.current.tier != .stable) {
+        webView!.isInspectable = true
+      }
+    }
```

Comment:
Comment originally posted when this was part of #11215:

> Potential improvement:  also add check for if `local` or `test`; those should be allowed to override .stable, since they're inherently less stable by virtue of being development versions.
> 
> I don't think our iOS engine's `Version` class yet exposes those as flags, though...